internal/commands/account/show.go

```diff
@@ -80,6 +81,11 @@ func (s *showCommand) ExecuteWithoutArguments(exec commands.Executor) (output.Ou
 			},
 		},
 	}, nil
+
+	return output.MarshaledWithHumanOutput{
```

Comment:
No change in JSON output as most recent fields (e.g., `load_balancers`) are missing from Go SDK at the moment